tests/running_the_node_and_interrupt.rs

```diff
@@ -26,9 +27,11 @@ fn running_the_node_works_and_can_be_interrupted() {
 	use nix::unistd::Pid;
 
 	fn run_command_and_kill(signal: Signal) {
-		let _ = fs::remove_dir_all("interrupt_test");
+		let tmpdir = tempdir().expect("coult not create temp dir");
+		let base_path = tmpdir.path().to_str().expect("path should consist of valid utf8 characters");
+
 		let mut cmd = Command::new(cargo_bin("polkadot"))
-			.args(&["--dev", "-d", "interrupt_test"])
+			.args(&["--dev", "-d", base_path])
```

Comment:
If you remove the last parameter here and call `arg` with `tmpdir.path()` you don't need to a string slice above.